### DISCUSSION TITLE
Cosmos chain onboarding proposal -- Neutron Chain

### DISCUSSION THREAD

**Discussion by NicholasDotSol** [wormhole-foundation/wormhole#3733] [Gateway]
### Website

https://www.neutron.org/

### Github

https://github.com/neutron-org/neutron

### Blog

https://blog.neutron.org/

### Documentation

https://docs.neutron.org/

### Community stats

- Twitter [41.5K Followers](https://twitter.com/Neutron_org/verified_followers)
- Varies telegram and builder chats

### Short Summary

Neutron is the first interchain secured smart contract platform on Cosmos. By granting smart contracts the powers of an appchain, Neutron enables developers to build breakthrough applications and onboard users from anywhere.

### Why should Wormhole add this chain?

- Light lift for Guardians - Guardians are not required to run a full node for Neutron - connectivity will be via a Gateway IBC connection.
- In order to enable cross-chain message passing required for Lido wstETH bridge aggregation and other token bridging.
- Other bridges already active: Axelar, Hyperlane
- Dapps: Home of Astroport, Mars, Nexus and more
- [43mm TVL](https://defillama.com/chain/Neutron)

### Technology and Features

- Built using Cosmos SDK
- Interhcain Accounts
- Interhcain queries
- Cron module for granular access to chain-specific functionality

### Native token details

1,000,000,000 max supply
used for governance and transaction fees
https://www.coingecko.com/en/coins/neutron

### Technical requirements for running a node

None - [chain] will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).

### IBC Relaying Strategy

Relaying will be handled by cryptoCrew: an external dedicated team. They already handle other connections and maintain great uptime

**Comment by ifedbro** [comment]
thanks
